src/badges/src/Badge.js

```diff
@@ -33,7 +33,7 @@ class Badge extends PureComponent {
   static styles = {
     display: 'inline-block',
     boxSizing: 'border-box',
-    height: 16,
+    minHeight: 16,
```

Comment:
What do you think of using `lineHeight` or flex to solve for this?

Reply:
Hmm, messing with line hight (setting it to 1.1) worked great for where there was a sup/sub tag...but then it was off centered without those tags. I'm actually in favor of closing this PR and having the user pass down the line height style for when they want to use a sup/sub tag. What do you think?